docs/help-center/using-famedly/notes.mdx

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_label: Notes
+title: Notes
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Notes</h2>
+    <p class="hero__subtitle">Save important messages, media and even message yourself!</p>
+  </div>
+</div>
+
+## Add to Notes
+
+You can save messages and media from other conversations by adding it to <b>Notes</b>.
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Move the cursor over the message you want to save.</li>
+    <li>Click on it and an action list will pop up.</li>
+    <li>Now, choose <b>Add to Notes</b> from the list.</li>
+    <li>The message will be saved in a private chat named <b>Notes</b>.</li>
+    <li>You can click on the <b>Go to Notes</b> button that is shown right after, to check.</li>
```

Comment:
I'd suggest to add a description where the button is located (at the bottom of the window) and maybe that it will disappear after some seconds so that people won't desperately search for it afterwards.